packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -88,23 +88,23 @@ const fetchLinkSuggestions = async (
 		type = undefined,
 		subtype = undefined,
 		page = undefined,
-		perPage = isInitialSuggestions ? 3 : 20,
+		perPage = 20,
 	} = searchOptions;
 
 	const { disablePostFormats = false } = settings;
 
 	/** @type {Promise<WPLinkSearchResult>[]} */
 	const queries = [];
 
-	if ( ! type || type === 'post' ) {
+	if ( isInitialSuggestions ) {
 		queries.push(
 			apiFetch( {
 				path: addQueryArgs( '/wp/v2/search', {
 					search,
 					page,
 					per_page: perPage,
 					type: 'post',
-					subtype,
```

Comment:
As state in the PR description this is a change across WP Core in how default links behave. This also applies to any 3rd party consumers of the component which utilise the `fetchLinkSuggestions` function from `@wordpress/core`.

Now all initial suggestions are forced to `Page`. There may be consumers of this component for whom that change is not what they wish and I believe it represents a potentially breaking change.

We should be absolutely certain this is what we want for all consumers before we merge this change.

Reply:
The problem with the way current initial suggestions work is that they are not meaningful. It would be an enhancement in terms of UX if the initial suggestions would always list pages. Ideally the initial suggestion query should be open to 3rd parties so they can fit their needs.

As it is now we show "a bit of everything" which doesn't really serve any purspose.